tests/func/test_remote.py

```diff
@@ -115,6 +115,18 @@ def test_show_default(dvc, capsys):
     assert out == "foo\n"
 
 
+def test_list_shows_default(dvc, capsys):
+    default_remote = "foo"
+    other_remote = "bar"
+    assert main(["remote", "add", default_remote, "s3://bucket/name"]) == 0
+    assert main(["remote", "add", other_remote, "s3://bucket/name"]) == 0
+    assert main(["remote", "default", default_remote]) == 0
+    assert main(["remote", "list"]) == 0
+    out, _ = capsys.readouterr()
+    assert "* foo" in out
+    assert "\nbar" in out
```

Comment:
```suggestion
    assert out == """\
* foo
  bar
"""
```

Perhaps you can make this test a bit more stricter.

Reply:
Fair point, I've committed a change now in 574afc7eedc0ed00f46c3773e9b1dd01ac365a5d 
I used the function (`escape_ansi`) which I saw was used in some of the other tests to get the "plain text" from the output (as the default remote now has styling) and then asserted both lines. Hopefully that looks alright?